webviz_subsurface/plugins/_parameter_analysis/parameter_analysis.py

```diff
@@ -0,0 +1,168 @@
+from typing import Optional
+
+import dash_core_components as dcc
+from webviz_config import WebvizPluginABC
+from webviz_config.webviz_assets import WEBVIZ_ASSETS
+
+import webviz_subsurface
+from .views import main_view
+from .models import ParametersModel, SimulationTimeSeriesModel, EnsembleSetModel
+from .controllers import (
+    property_qc_controller,
+    property_response_controller,
+)
+from .data_loaders import read_csv
+
+
+class ParameterAnalysis(WebvizPluginABC):
+    """This plugin visualizes parameter distributions and statistics. /
+    for FMU ensembles, and can be used to investigate parameter correlations /
+    on reservoir simulation time series data.
+
+---
+
+**Input data can be provided in two ways: Aggregated or read from ensembles stored on scratch.**
+
+**Using aggregated data**
+* **`csvfile_parameters`:** Aggregated `csv` file with `REAL`, `ENSEMBLE` and parameter columns. \
+    (absolute path or relative to config file).
+* **`csvfile_smry`:** (Optional) Aggregated `csv` file for volumes with `REAL`, `ENSEMBLE`, `DATE` \
+    and vector columns (absolute path or relative to config file).
+
+**Using raw ensemble data stored in realization folders**
+* **`ensembles`:** Which ensembles in `shared_settings` to visualize.
+* **`column_keys`:** List of vectors to extract. If not given, all vectors \
+    from the simulations will be extracted. Wild card asterisk `*` can be used.
+* **`time_index`:** Time separation between extracted values. Can be e.g. `monthly` (default) or \
+    `yearly`.
+
+**Common settings for all input options**
+* **`drop_constants`:** Bool used to determine if constant parameters should be dropped. \
+    Default is True.
+
+---
+
+!> For smry data it is **strongly recommended** to keep the data frequency to a regular frequency \
+(like `monthly` or `yearly`). This applies to both csv input and when reading from `UNSMRY` \
+(controlled by the `sampling` key). This is because the statistics and fancharts are calculated \
+per DATE over all realizations in an ensemble, and the available dates should therefore not \
+differ between individual realizations of an ensemble.
+
+?> Vectors that are identified as historical vectors (e.g. FOPTH is the history of FOPT) will \
+be plotted together with their non-historical counterparts as reference lines.
+
+**Using simulation time series data directly from `.UNSMRY` files**
+
+!> Parameter values are extracted automatically from the `parameters.txt` files in the individual
+realizations if you have defined `ensembles`, using the `fmu-ensemble` library.
+
+!> The `UNSMRY` files are auto-detected by `fmu-ensemble` in the `eclipse/model` folder of the \
+individual realizations. You should therefore not have more than one `UNSMRY` file in this \
+folder, to avoid risk of not extracting the right data.
+
+**Using aggregated data**
+
+?> Aggregated data may speed up the build of the app, as processing of `UNSMRY` files can be \
+slow for large models.
+
+"""
```

Comment:
Docstrings can now be indented after https://github.com/equinor/webviz-config/pull/275.